### PR TITLE
cmd/bosun: clear body and subject on unknown

### PR DESCRIPTION
**Pull request by mjibson**

### CODE REVIEW

File: cmd/bosun/sched/check.go [hunk 1]
```diff
@@ -129,6 +129,11 @@ func (s *Schedule) RunHistory(r *RunHistory) {
 		a := s.Conf.Alerts[ak.Name()]
 		wasOpen := state.Open
 		if event.Status > StNormal {
+			state.Subject = ""
```
Comment (captncraig) [inline cmd/bosun/sched/check.go]: should this be `state.ClearTemplateData()` or something?

Comment (mjibson) [inline cmd/bosun/sched/check.go]: I'd do that if you feel strongly. But it's only 5 lines and just needed in one place, so I don't think it needs to be a function.

Comment (captncraig) [inline cmd/bosun/sched/check.go]: fair enough

**Comment by captncraig** [comment]
LGTM

**Pull request merged**